home/views/admin.py

```diff
@@ -167,6 +167,11 @@ def post(self, request):
                 ProfessorCourse.objects.filter(professor__id=subject_id).update(professor=merge_target)
                 Review.unfiltered.filter(professor__id=subject_id).update(professor=merge_target)
                 Grade.unfiltered.filter(professor__id=subject_id).update(professor=merge_target)
+                
+                aliases = ProfessorAlias.objects.filter(name=merge_subject.name, professor=merge_target)
```

Comment:
filtering against `name` breaks as it isn't a valid field. Should be checking against `alias`.